package.json

```diff
@@ -1,5 +1,16 @@
 {
     "dependencies": {
-      "cosmiconfig": "8.0.0"
+        "commitlint": "*",
+        "conventional-changelog-conventionalcommits": "*",
+        "cosmiconfig": "8.0.0"
```

Comment:
@Mersho WHY ARE YOU ADDING THIS NOW?????????? WE WERE ONLY TALKING ABOUT DEV DEPENDENCIES HERE! THE COMMIT MESSAGE ONLY TALKS ABOUT DEV DEPENDENCIES!!!!!!!!!!!!!!!!!!

Reply:
WHY ARE YOU MIXING AGAIN CHANGES UNRELATED???????? WHY IS THIS PRINCIPLE SO HARD TO UNDERSTAND??????????